contrib/platform/test/com/sun/jna/platform/unix/LibCTest.java

```diff
@@ -85,4 +85,34 @@ public void testGethostnameGetdomainname() {
         System.out.println("Domainname: " + domainname);
         assertTrue(domainname.length() > 0);
     }
+
+    @Test
+    public void testSystem(){
+        assertSuccessResult("system", LibC.INSTANCE.system("ls"));
+    }
+
+    @Test
+    public void testGetpid(){
+        assertTrue("getpid", LibC.INSTANCE.getpid() > 0);
+    }
+
+    @Test
+    public void testGetppid(){
+        assertTrue("getppid", LibC.INSTANCE.getppid() > 0);
+    }
+
+    @Test
+    public void testFork(){
+        int status = 0;
+        int pid = LibC.INSTANCE.fork();
+        if (pid == 0) // This is the child process
+            return;
+        else if (pid < 0) // Fork failed
+            status = -1;
+        else { // This is the parent process. Wait for the child to complete.
+            if (LibC.INSTANCE.waitpid(pid, status, 0) != pid)
```

Comment:
This function must be tested before, before using it in this test.
Gotta find a way of testing this it.

Reply:
You could share memory between the two processes and write to it.